### PR TITLE
⚡️ [SUPPORT] : Rework Market with ReactQuery in LLM + CVS v3 on LL

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### 📝 Description

Replace MarketProvider in LLM new Arch with ReactQuery hook like LLD

- Introducing  LLM
- Usage of [TanstackQuery](https://tanstack.com/query/latest/docs/framework/react/overview) (ReactQuery)
- Adding new part in Store dedicated to Market pages
- Remove old stuff from Settings store to new Market store
- Removal of MarketDataProvider from **LLC**

### ❓ Context

- **JIRA or GitHub link**: [LIVE-11751] [LIVE-12816] 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-11751]: https://ledgerhq.atlassian.net/browse/LIVE-11751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIVE-12816]: https://ledgerhq.atlassian.net/browse/LIVE-12816?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketDetail/components/MarketGraph/index.tsx [hunk 1]
```diff
@@ -1,43 +1,34 @@
...
+  range,
 }: {
   setHoverItem: (_: Item | null | undefined) => void;
-  chartRequestParams: SingleCoinProviderData["chartRequestParams"];
-  loading?: boolean;
-  loadingChart?: boolean;
+  isLoading?: boolean;
   refreshChart: (_: { range: string }) => void;
-  chartData: Record<string, [number, number][]>;
+  chartData?: Record<string, [number, number][]>;
```
Comment (LucasWerey) [inline apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketDetail/components/MarketGraph/index.tsx:22]: Maybe create a general type as you use it somewhere else. 
And maybe make [number,number] more explicit with a type chartPoint

**Review by gre — APPROVED** [review]

File: apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketDetail/index.tsx [hunk 2]
```diff
@@ -105,30 +110,33 @@ function View({
...
-                    {" "}
-                    -
-                  </Text>
-                )}
+            {!loading && (
+              <Flex justifyContent="center" alignItems="flex-start" pb={3}>
+                <Text variant="h1" mb={1}>
+                  {counterValueFormatter({
+                    currency: counterCurrency,
+                    value: hoveredItem?.value ? hoveredItem.value : price || 0,
```
Comment (LucasWerey) [inline apps/ledger-live-mobile/src/newArch/features/Market/screens/MarketDetail/index.tsx:118]: You can set by default price to 0 inside _const { price, priceChangePercentage } = dataCurrency || {};_

**Review by KVNLS — APPROVED** [review]
Great improvements!

**Review by cgrellard-ledger — APPROVED** [review]

**Comment by cgrellard-ledger** [comment]
LGTM !

**Pull request merged**